tutorials/real-time-analytics-transport.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Real-time analytics with Timescale Cloud and Grafana
+excerpt: Simulate an IOT dataset in your Timescale Cloud service
```

Comment:
Is this the right excerpt? The excerpt should contain the main keyword (analytics) or some variation of it. For example, "Simulate and analyze an IoT dataset in your Timescale Cloud service"

Reply:
Nice spot. updated.